src/store/utility.ts

```diff
@@ -404,6 +404,14 @@ export function buildAudioFileNameFromRawData(
   });
 }
 
+export async function generateUniqueId(serializable: unknown) {
```

Comment:
便利ではあるのですが、用途が限定されているので現状は`src/store/audioGenerate.ts`内のプライベート関数にしておく方が良いかもと思いました！

Reply:
#1475 にて，この部分を分離し`utility.ts`に分離する方針が提案されていたのでそのようにしたのですが，将来これを別の場所で（キャッシュのキーとして）使うことはないのでしょうか．

Reply:
#1475 にてこの関数を`utility.ts`に分離するように提案されていたのでそのようにしたのですが，この関数は以降も別の場所では使われないのでしょうか．

Reply:
あ、失礼しました！
この関数が保存とかにも利用され始めると、マイグレーションとか考える必要が出てきて大変そうだなと思った感じです。
大丈夫ですかね…。

ちなみに既にsong側でも同じコードを使ってる部分があるかもです。uniqueとかでコード全体検索すると見つかるかも。

Reply:
この関数はメモリ上のキャッシュキーを作るためのものであると理解しています（その旨をコメントに書くべきかもしれません）ので、一意なキーをephemeralに生成できればいいのだと思っています。そもそも不可逆であり、保存に利用すべきではないと思います。

Reply:
一時的なキー、なるほどです！！
この意図を関数名とドキュメントに含めれば問題ないと感じました。ちょっと不格好ですが、こんな感じでどうでしょう？

```suggestion
/**
 * オブジェクトごとに一意なキーを作る。
 * 一時的な利用のみを想定しているため、保存に利用すべきではない。
 */
export async function generateTempUniqueId(serializable: unknown) {
```
ephemeralはちょっと馴染みがない方が多そうで、ぱっとわかるTempにしてみました。